packages/server/src/sdk/app/rows/search/internal.ts

```diff
@@ -0,0 +1,246 @@
+import { context } from "@budibase/backend-core"
+import env from "../../../../environment"
+import { fullSearch, paginatedSearch } from "./internalSearch"
+import {
+  InternalTables,
+  getRowParams,
+  DocumentType,
+} from "../../../../db/utils"
+import { getGlobalUsersFromMetadata } from "../../../../utilities/global"
+import { outputProcessing } from "../../../../utilities/rowProcessor"
+import { Ctx, Database, Row, UserCtx } from "@budibase/types"
+import { cleanExportRows } from "../utils"
+import {
+  Format,
+  csv,
+  json,
+  jsonWithSchema,
+} from "../../../../api/controllers/view/exporters"
+import { apiFileReturn } from "../../../../utilities/fileSystem"
+import * as userController from "../../../../api/controllers/user"
+import * as inMemoryViews from "../../../../db/inMemoryView"
+import {
+  migrateToInMemoryView,
+  migrateToDesignView,
+  getFromDesignDoc,
+  getFromMemoryDoc,
+} from "../../../../api/controllers/view/utils"
+
+export async function search(ctx: Ctx) {
```

Comment:
As above comments, again would prefer if we had common types for these SDK functions.